backend/src/v5/routes/teamspaces/projects/models/common/tickets.js

```diff
@@ -599,6 +600,12 @@ const establishRoutes = (isFed) => {
 	 *         required: false
 	 *         schema:
 	 *           type: string
+	 *       - name: query
+	 *         description: Query string that defies tickets to be included in the response
```

Comment:
we're going to need to give our users a bit more help than this. bear in mind any integrators will need to have access to info on how to construct this string.

So I think either we lay out the schema in swagger, or link them to a README in github (bit like the chat event README)

Reply:
I think we're missing limit and skip?